site/config/target_env.md

```diff
@@ -0,0 +1,93 @@
+```
+The `<target-name>` value corresponds to the value of the `-target` argument on the tool's command line. The WLS installation includes two pre-defined targets:
+ - [Oracle Weblogic Server Kubernetes Operator](#the-oracle-weblogic-server-kubernetes-operator-target) (named `k8s`)
+ - [Verrazzano](#the-verrazzano-target) (named `vz`). 
+
+You can define a new or extended target environment with a new `target-name` in the above location, or using a [Custom Configuration](custom_configuration.md) directory, such as `$WDT_CUSTOM_CONFIG/target/<my-target-name>/target.json`. 
+
+Here is an example of a target environment file:
+```
+{
+  "model_filters" : {
+        "discover": [
+          { "name": "k8s_prep", "path": "@@TARGET_CONFIG_DIR@@/k8s_operator_filter.py" }
+        ]
+      },
+  "variable_injectors" : {"PORT": {},"HOST": {},"URL": {}},
+  "validation_method" : "lax",
+  "credentials_method" : "secrets",
+  "additional_output" : "vz"
+}
+```
+Each of the fields in this example is optional, and can be customized.
+
+#### `model_filters`
+
+This field specifies the filters to be applied to the resulting model. This follows the same format and rules as the [Model Filters](../tool_filters.md) configuration. The `discover` type should always be used here.
+
+The `@@TARGET_CONFIG_DIR@@` token can be used to indicate that the specified filter is in the same directory as the target configuration file.  
+
+#### `variable_injectors`
+
+This field specifies the variable injectors to be applied to the resulting model. This follows the same format and rules as the [Variable Injectors](../variable_injection.md) configuration.
+
+#### `validation_method`
+
+This field can be used to set the validation level for the resulting model. Only the value `lax`is currently supported. With `lax` validation, variables and Kubernetes secrets referenced in the resulting model do not need to be available when the model is created.
+
+#### `credentials_method`
+
+This field specified how credentials in the model should be handled. There are two values available:
+- `secrets` - the credentials in the model are replaced with references to Kubernetes secrets, and a UNIX script to create those secrets is produced.
+- `config_override_secrets` - the credentials in the model are replaced with placeholder values, such as `password1`, and a UNIX script to create corresponding Kubernetes secrets is produced.
+
+In both these cases, the script to create the Kubernetes secrets is written to `<output-directory>/create_k8s_secrets.sh`. You will need to update this scripts with credential values before executing
+
+#### `additional_output`
+
+This field can be used to create additional output for use in the target environment. The key `vz` is currently the only valid value for this field. It indicates that the tool should create additional Kubernetes resource files for use with the Verrazzano product.
+
+### Pre-configured Target Environments
```

Comment:
Pre-configured Target Environments -> You need at least one sentence after a heading.